ci/README.md

```diff
@@ -316,4 +316,50 @@ BIGTABLE_SERVICE_URL=$(gcloud run services describe \
     --format="value(status.url)" \
     gcf-tutorial-cloud-bigtable)
 curl -H "projectID: ${GOOGLE_CLOUD_PROJECT}" -H "instanceID: test-instance-0" -H "tableID: test-table" "${BIGTABLE_SERVICE_URL}"
-```
\ No newline at end of file
+```
+
+### Spanner
+
+Create the instance:
+
+```shell
+gcloud --project="${GOOGLE_CLOUD_PROJECT}" spanner instances create test-instance-0 \
+    --config="regional-us-central1" \
+    --description="Test instance for CI builds" \
+    --nodes=1
+```
+
+To populate the database, use the spanner examples from the C++ client library:
+
+```shell
+(
+    cd $HOME/google-cloud-cpp
+    bazel build //google/cloud/spanner/samples:samples
```

Comment:
Opt: `run` will `build`, so you could omit this step.

Reply:
Dont.